test/BUILD.gn

```diff
@@ -279,6 +279,13 @@ test("brave_browser_tests") {
     ":brave_browser_tests_deps",
     ":browser_tests_runner",
   ]
+
+  if (brave_rewards_enabled) {
```

Comment:
we don't want bat-native-ledger to be added as a dep for anything outside of brave rewards component